python/python/bystro/supervised_ppca/gf_generative_pt.py

```
@@ -606,6 +606,71 @@ def _store_instance_variables(  # type: ignore[override]
+        S : NDArray,(N_samples,n_components)
+            The factor estimates
+        """
+        if self.W_ is None:
+            raise ValueError("Model has not been fit yet")
+
+        if sherman_woodbury is False:
+            prec = self.get_precision()
+            coefs = np.dot(self.W_, prec)
+        else:
+            M = np.dot(self.W_, self.W_.T) + np.eye(self.n_components) * self.sigma2_
+            coefs = np.dot(self.W_.T, la.inv(M)).T
```

Ruff

Undefined name `la`

(F821)